Formula/zork.rb

```diff
@@ -0,0 +1,21 @@
+class Zork < Formula
+  desc "The Dugeon modified from FORTRAN to C"
+  homepage "https://github.com/devshane/zork"
+  url "https://github.com/devshane/zork/archive/v1.0.0.tar.gz"
+  version "1.0.0"
+  sha256 "ceedb436cf552bd8ca39abd3e85bae986c3f5df6f31c1bf2806d1b4c8509da67"
+  head "https://github.com/devshane/zork.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "make", "LIBDIR=#{lib}", "BINDIR=#{bin}"
+    bin.install "zork"
+    lib.install "dtextc.dat"
+    man6.install "dungeon.6"
```

Comment:
In my testing, using `make install` bombs because it tries to `cp` these files to `/usr/games` and `/usr/games/lib`: https://github.com/devshane/zork/blob/v1.0.0/Makefile#L68-L70

Is there a better way to handle that?